pkg/loop/internal/keystore/types.go

```diff
@@ -0,0 +1,30 @@
+package keystore
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+)
+
+// Methods This interface contains all the functionalities of the GRPC layer of the LOOPP keystore
+type Methods interface {
```

Comment:
This isn't going to be super helpful as feedback but Methods is a confusing name IMO -- what are you describing here? It sounds like it's a public interface of some kind, so maybe keystore.Public would be better?

Reply:
Renamed the interface to illustrate that this is only the GRPC service [`9bc2713` (#837)](https://github.com/smartcontractkit/chainlink-common/pull/837/commits/9bc2713bc46d6518a758830828454e7077039dbe)